build.gradle

```diff
@@ -254,7 +258,14 @@ testClusters.integTest {
             return new RegularFile() {
                 @Override
                 File getAsFile() {
-                    return fileTree("src/test/resources/job-scheduler").getSingleFile()
+                    project.mkdir js_resource_folder
+                    ant.get(src: 'https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/'
+                            + opensearch_no_snapshot
+                            + '/latest/linux/x64/builds/opensearch/plugins/opensearch-job-scheduler-'
```

Comment:
I see we use hard code `x64` here, not sure if it will impact CI/IT workflow for other platform, @peterzhuamazon any suggestion?

Reply:
Since github actions offer default runner are x64 I see no problem running on hardcoded x64.
Jenkins are testing both x64 and arm64 tho after each build.
If you really want arm64 to be on github action you need to manage self-hosted runners yourself tho.

Reply:
This will eventually become a problem, but not just because of x64. For example, windows. For now this is fine.